util/chplenv/utils.py

```diff
@@ -33,7 +33,7 @@ class CommandError(Exception):
     pass
 
 
-def run_command(command, stdout=True, stderr=False, cmd_input=None):
+def run_command_deluxe(command, stdout=True, stderr=False, cmd_input=None):
     """Command subprocess wrapper.
```

Comment:
We probably want to change the documentation here slightly to say what this does and probably point most people towards `run_command` instead.